scripts/main.js

```diff
@@ -1,5 +1,50 @@
 var helloCork = function() {
-  console.log('Hello Cork');
+  	console.log('Hello Cork');
 };
 
+var signedUrlUpcoming = "https://api.meetup.com/cork-javascript-meetup/events?photo-host=public&page=20&sig_id=210143296&sig=d07ac4460173bc86aa05e0c5b65acd79b2b8ae1b";
+var signedUrlPast = "https://api.meetup.com/cork-javascript-meetup/events?photo-host=public&page=20&sig_id=210143296&status=past&sig=052868f6f4d1f5a525fa6ca28a74e771165ad4ea";
+
+var getUpcomingEvents = function() {
+	var call = $.ajax({
+		type: "GET",
+		dataType: "jsonp",
```

Comment:
No, I think to get CORS working then the user has to go through an OAUTH flow. This means the user will be redirected to a page where they are asked to authorize our application.  Let's stick with JSONP.